packages/shorebird_cli/test/src/extensions/version_test.dart

```diff
@@ -0,0 +1,38 @@
+import 'package:pub_semver/pub_semver.dart';
+import 'package:shorebird_cli/src/extensions/version.dart';
+import 'package:test/test.dart';
+
+void main() {
+  group('NonStrictParsing', () {
```

Comment:
```suggestion
  group('parsing', () {
```

since this group contains both strict and lax parsing shouldn't it be renamed?

Reply:
Yeah, this is a holdover from a previous extension name. Fixed